Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Kimai - Time tracking/Main application](https://hosted.weblate.org/projects/kimai/messages/).


It also includes following components:

* [Kimai - Time tracking/Reporting](https://hosted.weblate.org/projects/kimai/reporting/)

* [Kimai - Time tracking/Date-Picker](https://hosted.weblate.org/projects/kimai/daterangepicker/)

* [Kimai - Time tracking/Emails](https://hosted.weblate.org/projects/kimai/emails/)

* [Kimai - Time tracking/Invoice-Calculator](https://hosted.weblate.org/projects/kimai/invoice-calculator/)

* [Kimai - Time tracking/User feedback (Flash messages and Alerts)](https://hosted.weblate.org/projects/kimai/flashmessages/)

* [Kimai - Time tracking/Tasks](https://hosted.weblate.org/projects/kimai/tasks/)

* [Kimai - Time tracking/Teams](https://hosted.weblate.org/projects/kimai/teams/)

* [Kimai - Time tracking/Kiosk](https://hosted.weblate.org/projects/kimai/kiosk/)

* [Kimai - Time tracking/System configurations](https://hosted.weblate.org/projects/kimai/system-configuration/)

* [Kimai - Time tracking/Invoice-Templates](https://hosted.weblate.org/projects/kimai/invoice-templates/)

* [Kimai - Time tracking/Wizard](https://hosted.weblate.org/projects/kimai/wizard/)

* [Kimai - Time tracking/Plugins](https://hosted.weblate.org/projects/kimai/plugins/)

* [Kimai - Time tracking/Validations](https://hosted.weblate.org/projects/kimai/validations/)

* [Kimai - Time tracking/Export](https://hosted.weblate.org/projects/kimai/export/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/kimai/messages/horizontal-auto.svg)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/kimai/kimai?pullRequest=5306) <br/>All committers have signed the CLA.

weblate committed:
Update translation files

Updated by "Remove blank strings" hook in Weblate.

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translated using Weblate (Estonian)

Currently translated at 5.8% (1 of 17 strings)

Translated using Weblate (Tamil)

Currently translated at 97.4% (38 of 39 strings)

Translated using Weblate (Slovenian)

Currently translated at 89.7% (35 of 39 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 89.7% (35 of 39 strings)

Translated using Weblate (Ukrainian)

Currently translated at 97.4% (38 of 39 strings)

Translated using Weblate (German (Switzerland))

Currently translated at 48.7% (19 of 39 strings)

Translated using Weblate (Faroese)

Currently translated at 46.1% (18 of 39 strings)

Translated using Weblate (Persian)

Currently translated at 61.5% (24 of 39 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 51.2% (20 of 39 strings)

Translated using Weblate (Croatian)

Currently translated at 97.4% (38 of 39 strings)

Translated using Weblate (Greek)

Currently translated at 51.2% (20 of 39 strings)

Translated using Weblate (Finnish)

Currently translated at 51.2% (20 of 39 strings)

Translated using Weblate (Portuguese)

Currently translated at 92.3% (36 of 39 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 89.7% (35 of 39 strings)

Translated using Weblate (Vietnamese)

Currently translated at 51.2% (20 of 39 strings)

Translated using Weblate (Turkish)

Currently translated at 97.4% (38 of 39 strings)

Translated using Weblate (Swedish)

Currently translated at 92.3% (36 of 39 strings)

Translated using Weblate (Slovak)

Currently translated at 61.5% (24 of 39 strings)

Translated using Weblate (Russian)

Currently translated at 51.2% (20 of 39 strings)

Translated using Weblate (Romanian)

Currently translated at 61.5% (24 of 39 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 97.4% (38 of 39 strings)

Translated using Weblate (Polish)

Currently translated at 89.7% (35 of 39 strings)

Translated using Weblate (Dutch)

Currently translated at 92.3% (36 of 39 strings)

Translated using Weblate (Korean)

Currently translated at 51.2% (20 of 39 strings)

Translated using Weblate (Japanese)

Currently translated at 5.1% (2 of 39 strings)

Translated using Weblate (Italian)

Currently translated at 97.4% (38 of 39 strings)

Translated using Weblate (Hungarian)

Currently translated at 41.0% (16 of 39 strings)

Translated using Weblate (Hebrew)

Currently translated at 97.4% (38 of 39 strings)

Translated using Weblate (French)

Currently translated at 92.3% (36 of 39 strings)

Translated using Weblate (Basque)

Currently translated at 17.9% (7 of 39 strings)

Translated using Weblate (Spanish)

Currently translated at 97.4% (38 of 39 strings)

Translated using Weblate (Esperanto)

Currently translated at 46.1% (18 of 39 strings)

Translated using Weblate (German)

Currently translated at 97.4% (38 of 39 strings)

Translated using Weblate (Danish)

Currently translated at 89.7% (35 of 39 strings)

Translated using Weblate (Czech)

Currently translated at 92.3% (36 of 39 strings)

Translated using Weblate (Arabic)

Currently translated at 53.8% (21 of 39 strings)

Translated using Weblate (English)

Currently translated at 100.0% (39 of 39 strings)

Translated using Weblate (Estonian)

Currently translated at 100.0% (4 of 4 strings)

Translated using Weblate (Estonian)

Currently translated at 18.7% (91 of 485 strings)

Translated using Weblate (Estonian)

Currently translated at 30.7% (4 of 13 strings)

Translated using Weblate (Estonian)

Currently translated at 17.5% (85 of 485 strings)

Translated using Weblate (Estonian)

Currently translated at 23.0% (3 of 13 strings)

Translated using Weblate (Estonian)

Currently translated at 17.3% (84 of 485 strings)

Translated using Weblate (Estonian)

Currently translated at 15.3% (2 of 13 strings)

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translated using Weblate (Estonian)

Currently translated at 14.4% (70 of 485 strings)

Translated using Weblate (Estonian)

Currently translated at 14.4% (70 of 485 strings)

Translated using Weblate (Estonian)

Currently translated at 7.6% (1 of 13 strings)

Translated using Weblate (Estonian)

Currently translated at 0.6% (3 of 485 strings)

Translated using Weblate (Estonian)

Currently translated at 0.6% (3 of 485 strings)

Translated using Weblate (Estonian)

Currently translated at 0.2% (1 of 485 strings)

Translated using Weblate (Estonian)

Currently translated at 0.2% (1 of 485 strings)

Translated using Weblate (Estonian)

Currently translated at 0.0% (0 of 485 strings)

Added translation using Weblate (Estonian)

Translated using Weblate (Estonian)

Currently translated at 100.0% (10 of 10 strings)

Added translation using Weblate (Estonian)

Translated using Weblate (Estonian)

Currently translated at 90.0% (9 of 10 strings)

Added translation using Weblate (Estonian)

Translated using Weblate (Estonian)

Currently translated at 80.0% (8 of 10 strings)

Added translation using Weblate (Estonian)

Translated using Weblate (Estonian)

Currently translated at 50.0% (5 of 10 strings)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Translated using Weblate (Estonian)

Currently translated at 40.0% (4 of 10 strings)

Added translation using Weblate (Estonian)

Translated using Weblate (Estonian)

Currently translated at 30.0% (3 of 10 strings)

Added translation using Weblate (Estonian)

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Added translation using Weblate (Estonian)

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translated using Weblate (Estonian)

Currently translated at 10.0% (1 of 10 strings)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Translated using Weblate (Estonian)

Currently translated at 100.0% (2 of 2 strings)

Translated using Weblate (Estonian)

Currently translated at 50.0% (1 of 2 strings)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Translated using Weblate (Tamil)

Currently translated at 100.0% (39 of 39 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (27 of 27 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (78 of 78 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kevin Papst <kevin@kevinpapst.de>
Co-authored-by: Languages add-on <noreply-addon-languages@weblate.org>
Co-authored-by: Prefill add-on <noreply-addon-prefill@weblate.org>
Co-authored-by: TaaviLepik <info@brunex.ee>
Co-authored-by: தமிழ்நேரம் <anishprabu.t@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/kimai/daterangepicker/
Translate-URL: https://hosted.weblate.org/projects/kimai/daterangepicker/et/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/et/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/et/
Translate-URL: https://hosted.weblate.org/projects/kimai/kiosk/et/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/et/
Translate-URL: https://hosted.weblate.org/projects/kimai/plugins/et/
Translate-URL: https://hosted.weblate.org/projects/kimai/system-configuration/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/tasks/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/cs/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/de/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/de_CH/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/el/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/en/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/eo/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/es/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/eu/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/fa/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/fi/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/fo/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/he/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/hr/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/hu/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/it/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/ja/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/ko/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/nl/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/pl/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/pt/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/ro/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/ru/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/sk/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/sl/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/sv/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/tr/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/uk/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/vi/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/zh_Hant/
Translation: Kimai - Time tracking/Date-Picker
Translation: Kimai - Time tracking/Emails
Translation: Kimai - Time tracking/Invoice-Calculator
Translation: Kimai - Time tracking/Kiosk
Translation: Kimai - Time tracking/Main application
Translation: Kimai - Time tracking/Plugins
Translation: Kimai - Time tracking/System configurations
Translation: Kimai - Time tracking/Tasks
Translation: Kimai - Time tracking/Validations

kevinpapst committed:
Delete translations/export.et.xlf

kevinpapst committed:
Delete translations/flashmessages.et.xlf

kevinpapst committed:
Delete translations/invoice-renderer.et.xlf

kevinpapst committed:
Delete translations/reporting.et.xlf

kevinpapst committed:
Delete translations/system-configuration.et.xlf

kevinpapst committed:
Delete translations/teams.et.xlf

kevinpapst committed:
Delete translations/wizard.et.xlf

kevinpapst committed:
Delete translations/validators.et.xlf

kevinpapst committed:
Delete translations/tasks.et.xlf
